organization/general_rules_and_guidelines.rst

```diff
@@ -0,0 +1,71 @@
+General rules and guidelines
+============================
+
+This page contains guidelines relevant for contributing to any area of Godot.
+
+When contributing to any particular area, please also refer to its respective guidelines.
+
+Respect our Code of Conduct
+-------------------------------
+
+When you contribute to Godot, we expect that you respect our `Code of Conduct <https://godotengine.org/code-of-conduct/>`__.
+
+.. _doc_licenses:
+
+Respect intellectual property
+-----------------------------
+
+You must be mindful of the intellectual property of anything you submit.
+
+If your contribution was completely authored by you, this is not a problem. You can submit your contribution
+without reading further.
+
+However, any code, images, or ideas you took from somewhere else are "intellectual property" of the source
+you took it from.
+
+In this case, you need to check the license of the intellectual property. Some are permissive enough to be compatible
+with Godot's `MIT license <https://github.com/godotengine/godot/blob/master/LICENSE.txt>`__. You must include this
+license in your contribution.
+Examples of acceptable licenses include Apache 2.0, BSD, MIT, ISC, and MPL 2.0.
+
+However, others are not compatible with Godot's license. This includes "copy-left" licenses like GPL or LGPL,
+since these licenses effectively disallow static linking in proprietary software
+(which Godot is distributed as in most exported projects).
+This also applies for libraries that are only linked in the editor.
+
+Another example of incompatible licenses you should be aware of is "source-available" code.
+Most notably, you cannot submit code or ideas from proprietary game engines like Unreal or Unity.
+This includes using their code as a reference. We strongly recommend against reading any "source-available" code before
+making contributions to Godot, as this may impact your ability to contribute.
```

Comment:
Other engines too may (and do) use freely available code as reference for their implementations. So this part can feel a bit vague...

I know what this means but "as this may impact your ability to contribute." implies we come to their house and break their knees and it's making me chuckle a bit.
```suggestion
"Source-available" is **not** "open-source". For example, you cannot submit code or ideas exclusively from proprietary game engines like Unreal or Unity.
This includes using their code as a reference. We strongly recommend against reading any "source-available" code before
making contributions to Godot, as this may impact your ability to contribute.
```

Reply:
> I know what this means but "as this may impact your ability to contribute." implies we come to their house and break their knees and it's making me chuckle a bit.

"Read proprietary code at your own peril. You have been warned."